### PR TITLE
Solution next happy number added

### PR DESCRIPTION
**Pull request by Vipullakum007**
## Related Issue

i want to add solution of "next happy number " problem https://github.com/CodeHarborHub/codeharborhub/issues/389
fixes : https://github.com/CodeHarborHub/codeharborhub/issues/389

## Description

i have added solution of "next happy number " problem

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub/assets/127659443/484cb717-080a-46b1-a8fe-1e9ed987ad00

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/gfg-solutions/0004-fibbonacci-sum.md [hunk 1]
```diff
@@ -267,4 +265,7 @@ By leveraging the properties of Fibonacci numbers and matrix multiplication, we
 - **GeeksforGeeks Problem:** [Geeks for Geeks Problem](https://www.geeksforgeeks.org/problems/fibonacci-sum/0)
 - **Solution Link:** [Fibonacci Sum on Geeks for Geeks](https://www.geeksforgeeks.org/problems/fibonacci-sum/0)
 - **Authors GeeksforGeeks Profile:** [Vipul](https://www.geeksforgeeks.org/user/lakumvipwjge/)
+<<<<<<<<< Temporary merge branch 1
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/0004-fibbonacci-sum.md:268]: remove 

```md
<<<<<<<<< Temporary merge branch 1

=========
>>>>>>>>> Temporary merge branch 2
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
now remove code based on suggestions

**Comment by Vipullakum007** [comment]
hey @Ajay-Dhangar , updated as per your suggestion you can merge it

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Vipullakum007** [comment]
hey @Ajay-Dhangar  , you add level3  easy problem "How many X's " , "Remove Duplicate" , then why level 2  by  @manishh12  for "my meduim problems can you make level3.

**Pull request merged**